src/ui/public/vis/editors/default/agg_params.js

```diff
@@ -28,6 +28,29 @@ import { uiModules } from '../../../modules';
 import { documentationLinks } from '../../../documentation_links/documentation_links';
 import aggParamsTemplate from './agg_params.html';
 import { aggTypeFilters } from '../../../agg_types/filter';
+import { editorConfigProviders } from '../config/editor_config_providers';
+
+// TODO: This is just an example how to use this, and will be removed later.
```

Comment:
Does this still need to be here? It does seem useful, so maybe we should remove the `TODO` annotation just change this comment to be `This is an example of how to use this.`?

Reply:
Wait why is this still there? Oh yeah I forgot to push my last commit from yesterday :D